### ISSUE TITLE
i18n better integration

### ISSUE THREAD

**Issue by ASchmidt84** [sfxcode/sapphire-core#18]
Hi Tom,
I have an idea for better integration of i18n. The resource is loaded in the controller. That is fine. The key could contain variable placeholders. So MessageFormat can used. But I think it is redundant to use that. I suggest this additional class:
```
class Messages(bundle: ResourceBundle) {

  def getString(key: String, params: AnyRef*): String = {
    Try(bundle.getString(key)).map{f ⇒
      MessageFormat.format(f,params:_*)
    }.getOrElse(s"!!--$key--!!")
  }

}
```

and a method in ViewController 
```
private lazy val msg: Messages = new Messages(resources)

  def i18n(key: String, params: AnyRef*): String = {
    msg.getString(key,params)
  }
```

So it is a little bit easier to translate.

**Comment by sfxcode** [comment]
Will be released in the next version

**Comment by sfxcode** [comment]
Implemented in the 1.6.6 Release
